SofaKernel/framework/framework_test/core/DataEngine_test.cpp

```diff
@@ -112,10 +158,17 @@ struct DataEngine_test: public BaseTest
 
 };
 
+
+
 // Test
 TEST_F(DataEngine_test, testTrackedData )
 {
-    this->testTrackedData();
+    this->testTrackedData<TestEngine>(this->engine);
+}
+
+TEST_F(DataEngine_test, testSimpleEngine )
+{
+    this->testTrackedData<SimpleTestEngine>(this->simpleEngine);
```

Comment:
You don't test the case where the Data has been changed?

Reply:
Eh.. Nah I didn't ^^" Will do.
All I did in the test is replicate the already existing tests on the SimpleDataEngine (lazy version...)
I will add tests for the update() method

Reply:
perfect, thanks.

Reply:
Actually, I think the test you are mentioning is performed in the update() method, with if (m_dataTracker.isDirty(input)).
Can you confirm that this is what you meant? if so, I think actually that there's no need for additional testing no?

Reply:
my bad.